packages/magento-product-configurable/ConfigurableProductAddToCart/ConfigurableProductAddToCart.tsx

```diff
@@ -162,18 +162,21 @@ export function ConfigurableProductAddToCart(props: ConfigurableProductAddToCart
           !data?.addProductsToCart?.user_errors?.length
         }
         variant='pill'
+        autoHide
         action={
-          <PageLink href='/cart' passHref>
-            <Button
-              id='view-shopping-cart-button'
-              size='medium'
-              variant='pill'
-              color='secondary'
-              endIcon={<IconSvg src={iconChevronRight} />}
-            >
-              <Trans id='View shopping cart' />
-            </Button>
-          </PageLink>
+          !isMobile && (
+            <PageLink href='/cart' passHref>
+              <Button
+                id='view-shopping-cart-button'
+                size='medium'
+                variant='pill'
+                color='secondary'
+                endIcon={<IconSvg src={iconChevronRight} />}
+              >
+                <Trans id='View shopping cart' />
+              </Button>
+            </PageLink>
+          )
```

Comment:
I like the old one better, new one seems change because we can with reduced value (no button)